elements/pfe-progress-indicator/src/pfe-progress-indicator.scss

```diff
@@ -0,0 +1,68 @@
+@import "../../pfe-sass/pfe-sass";
+
+$pfe-progress-indicator--border-light: 4px solid rgba(0, 0, 0, 0.25);
+$pfe-progress-indicator--border-dark: 4px solid rgba(0, 0, 0, 0.75);
+
+:host([hidden]) {
+  display: none;
+}
+
+:host([pfe-indeterminate]) {
+    animation: spin 1s linear infinite;
+    -webkit-animation:spin 1s linear infinite;
+    -moz-animation:spin 1s linear infinite;
+
+    border-bottom: $pfe-progress-indicator--border-light;
+    border-left: $pfe-progress-indicator--border-light;
+    border-right: $pfe-progress-indicator--border-light;
+    border-top: $pfe-progress-indicator--border-dark;
+    border-radius: 100%;
+    
+    display: inline-block;
+    height: 2rem;
+    margin: 0 auto;
+    position: relative;
+    width: 2rem;
+    vertical-align: middle;
+    visibility: visible;
+}
+
+@-ms-keyframes spin {
+    from {
+        -ms-transform: rotate(0deg);
+    } to {
+        -ms-transform: rotate(360deg);
+    }
+}
+@-moz-keyframes spin {
+    from {
+        -moz-transform: rotate(0deg);
+    } to {
+        -moz-transform: rotate(360deg);
+    }
+}
+@-webkit-keyframes spin {
+    from {
+        -webkit-transform: rotate(0deg);
+    } to {
+        -webkit-transform: rotate(360deg);
+    }
+}
+@keyframes spin {
+    from {
+        transform: rotate(0deg);
+    } to {
+        transform: rotate(360deg);
+    }
+}
+
+  .sr-only {
```

Comment:
There's already an sr-only mixin, you can replace this with:
```
.sr-only {
   @include pfe-sr-only;
}
```

Reply:
Awesome, incorporated.